From 4e797418017f58dc599bbe6ded325764802fbc44 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Mon, 3 Jul 2023 15:15:12 +0200
Subject: [PATCH 0343/1141] arm64: dts: st: enable crc on stm32mp257f-ev1 board

Set STM32 CRC peripheral enabled.

Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: Icc2f29f96cf349606a4c37753e158488e40704d7
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/316041
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 4 ++++
 1 file changed, 4 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -130,6 +130,10 @@
 	status = "okay";
 };
 
+&crc {
+	status = "okay";
+};
+
 &cryp1 {
 	status = "okay";
 };
